web-common/src/features/file-explorer/new-files.ts

```diff
@@ -116,6 +116,11 @@ export const ResourceKindMap: Record<
     baseName: "alert",
     extension: ".yaml",
   },
+  [ResourceKind.RefreshTrigger]: {
+    folderName: "triggers",
+    baseName: "refresh_trigger",
+    extension: ".yaml",
+  },
```

Comment:
This list is "user-facing kinds", and I wouldn't consider the refresh trigger user-facing. Instead, you could add `RefreshTrigger` to the exclusion list here:
```
export type UserFacingResourceKinds = Exclude<
  ResourceKind,
  ResourceKind.ProjectParser,
  ResourceKind.RefreshTrigger // Add this
>;
```

Reply:
thanks!